sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java

```diff
@@ -79,7 +79,6 @@ public Duration getDuration() {
      * Regions contacted for this request
      * @return set of regions contacted for this request
      */
-    @Beta(value = Beta.SinceVersion.V4_9_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     public Set<URI> getRegionsContacted() {
```

Comment:
can we postpone GAing the `getRegionsContacted()` API for now.
I suspect we may need to return a different type, e.g., Set<URL> etc.

Reply:
Also, the feedResponseDiagnostics doesnt return all the values for this and currently just null. So its worth waiting to fix that part too for GA

Reply:
@moderakh  - makes sense, reverted this API back to Beta.